readinglist/resource.py

```diff
@@ -140,16 +156,12 @@ def _extract_sorting(self, queryparams):
                 sorting.append((field, direction))
         return sorting
 
-    def __decode_filter_value(self, value):
-        """Converts string value to native python values."""
-        if value.lower() in ['on', 'true', 'yes', '1']:
-            value = True
-        elif value.lower() in ['off', 'false', 'no', '0']:
-            value = False
-        try:
-            return ast.literal_eval(value)
-        except ValueError:
-            return value
+    def merge_fields(self, stored, changes):
+        """Merge given request fields with stored fields"""
```

Comment:
Merge changes into current record fields ?

Shouldn't we use ``self.record`` here instead of passing ``stored`` ?